cli/src/cli_input/sub_command.rs

```diff
@@ -10,12 +10,14 @@ use super::{
 
 #[derive(Debug, Parser, strum::AsRefStr, strum::Display)]
 #[strum(serialize_all = "lowercase")]
-pub enum SubCommand {
+pub(crate) enum SubCommand {
     /// Graph branch management
     Branch(BranchCommand),
     /// Output completions for the chosen shell to use, write the output to the
     /// appropriate location for your shell
     Completions(CompletionsCommand),
+    /// List all third-party subcommands. To list first-party subcommands, run `grafbase help`.
+    ListThirdParty,
```

Comment:
is this needed? Why not show plugin commands by default when you run grafbase help?

Reply:
It's not possible because with clap, the help text template has to be known at compile time. Cargo has the same issue.

Reply:
And git.